vendor/es5-shim.js

```diff
@@ -0,0 +1,1660 @@
+/*!
```

Comment:
add this shim to ensure that test suite has all the tools that brisket expects where it runs - adding this to solve phantomjs may not support Function.prototype.bind (https://github.com/ariya/phantomjs/issues/10522)